MSAL/src/telemetry/MSALAggregatedDispatcher.m

```diff
@@ -17,24 +15,43 @@
 //
 // THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
 // IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
 // AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 // LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
-//
-//------------------------------------------------------------------------------
 
-#import "MSALTestAppTelemetryDispatcher.h"
+#import "MSALAggregatedDispatcher.h"
+#import "MSIDDefaultDispatcher+Internal.h"
+#import "MSIDTelemetryBaseEvent.h"
+#import "MSIDTelemetryEventStrings.h"
+#import "MSALTelemetryEventsObservingProxy.h"
+
+@interface MSALAggregatedDispatcher ()
```

Comment:
Do we even need this class? Doesn't seem to be doing much. Can we instead call MSALTelemetryEventsObservingProxy's functionality directly?

Reply:
Good point. Removed MSAL dispatchers.